examples/virtual_methods/main.rs

```diff
@@ -0,0 +1,20 @@
+mod base_button;
+mod derived_button;
+mod virtual_methods_window;
+
+use crate::virtual_methods_window::VirtualMethodsAppWindow;
+use gtk::prelude::*;
+
+fn main() {
+    let application = gtk::Application::new(
+        Some("com.github.gtk-rs.examples.virtual_methods"),
+        Default::default(),
+    );
+
+    application.connect_activate(|app| {
+        let win = VirtualMethodsAppWindow::new(app);
```

Comment:
This seems to complicate the example unnecessarily. We already have a composite template example, so here you could for simplicity just create a window manually with the button.

Or does this add something to the example?